internal/cmds/cmds.go

```
@@ -335,10 +340,20 @@ func checkAndSaveSeqNum(ctx log.Logger, seq int, mrseqPath string) (shouldExit b
 	return false, nil
 }
 
+// resetSeqNum deletes the seqNum file to reset the sequence number
+func resetSeqNum(ctx log.Logger, mrseqPath string) error {
+	ctx.Log("event", "resetting seqnum by deleting file", "path", mrseqPath)
```

Do we need to return the error? It's not captured above, and Guest Agent ignores any disable failures.

---

Got it, if GA ignores any disabled failures, then we should not return any error from here.